### PR TITLE
TextTemplate for Handlebars and Mustache

### PR DESCRIPTION
**Pull request by erbg**
Simple Node to create Text strings by using a Templating Engine. In this version Mustache and Handlebars is available.

### CODE REVIEW

File: packages/nodes-base/nodes/TextTemplate/TextTemplate.node.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		// Input data
+		const item = this.getInputData();
+
+		let body: IDataObject | string = item.json;
+
+		const template = this.getNodeParameter('template', '') as string;
+		const templateEngine = this.getNodeParameter('templateengine', '') as string;
+
+		let rend = "";
+		if (templateEngine == "handlebars") {
```
Comment (AxelRueweler) [inline packages/nodes-base/nodes/TextTemplate/TextTemplate.node.ts:65]: I think you meant templateEngine == "mustache"

Comment (erbg) [reply packages/nodes-base/nodes/TextTemplate/TextTemplate.node.ts]: Thank you for reviewing my code. You are right of course.

**Comment by t-knapp** [comment]
Is there a reason for the PR being closed w/o the Node available in version 0.113.0?

**Comment by Aracon** [comment]
Why the PR was closed?

**Pull request closed without merging**